tests/oneOfWeighted.test.js

```diff
@@ -4,30 +4,35 @@ const { diffBetween } = require('./utils')
 
 const DIFF_THRESHOLD = 0.065
 
-test(`averages to within ${
-  DIFF_THRESHOLD * 100
-}% of the given probabilities`, t => {
-  const n = 10000
-  let i = -1
+test(`averages to within ${DIFF_THRESHOLD * 100
+  }% of the given probabilities`, t => {
+    const n = 10000
+    let i = -1
 
-  const sums = {
-    red: 0,
-    green: 0,
-    blue: 0
-  }
+    const sums = {
+      red: 0,
+      green: 0,
+      blue: 0,
+      yellow: 0,
+      fuchsia: 0
+    }
 
-  const fn = oneOfWeighted([
-    [0.6, 'red'],
-    [0.1, 'green'],
-    [0.3, 'blue']
-  ])
+    const fn = oneOfWeighted([
+      [0.6, 'red'],
+      [0.2, 'green'],
+      [0.1, 'blue']
```

Comment:
@ctholho only after landing and attempting to release did I realise we need `,`'s here (sorry, embarassingly fictional does not yet have CI).

After fixing the syntax though, it seems as though the test fail. Your fix and the test's logic both seem correct though, so my gut feeling is there is something else (that was already) broken in `oneOfWeighted`'s implementation.

I unfortunately don't enough time right now to investigate further though, so for now I'll re-open the issue and revert this PR's changes. Feel free to pick it up from there, otherwise I'll look into it further when I some time.

Hope that makes sense!